code/modules/cargo/exports/weapons.dm

```diff
@@ -11,18 +11,18 @@
 	include_subtypes = TRUE
 
 /datum/export/weapon/knife
-	cost = 750
+	cost = 100
 	unit_name = "combat knife"
 	export_types = list(/obj/item/kitchen/knife/combat)
```

Comment:
Are there other (reasonable) avenues of obtaining this other than roundstart miners and ordering via crates that would justify a nerf? If not, I don't see why the large nerf.